cmd/eudico/subnet.go

```diff
@@ -271,7 +293,9 @@ var syncCmd = &cli.Command{
 		if err != nil {
 			return err
 		}
-		fmt.Fprintf(cctx.App.Writer, "Successfully started/stopped syncing with subnet %s \n", subnet)
+		if _, err := fmt.Fprintf(cctx.App.Writer, "Successfully started/stopped syncing with subnet %s \n", subnet); err != nil {
```

Comment:
I don't think this is right, we don't care about the Fprintf error, we can forget about it, right? (see also changes below)